cite.html

```diff
@@ -0,0 +1,178 @@
+        }
+
+        .editbox {
+            background-color: #5f79b8;
+            width: 75%;
+            padding: 10px;
+        }
+        
+        .note {
+            background: #c18166;
+            width: 75%;
+            font-size: 150%;
+	    margin: auto;
+	    border: 1px solid blue;
+            padding: 5px;
+	text-align: center;
+        }
+    </style>
+</head>
+
+<body>
+
+    <p class="note">NOTE: a link with your selection is available at:<br/>
+        <a style="font-size:75%" id="onlineurl" href="https://geodynamics.github.io/aspect/cite.html">https://geodynamics.github.io/aspect/cite.html</a>
+    </p>
+
+<script src="database.js"></script>
+        
+    <script type="text/javascript">
+        var checkboxes = ["fs", "melt","particles","dg"];
+
+	var x= 'abc\"def';
+
+
+        var intext = "Computations were done using the ASPECT code version <i>VERSION</i>, see \n\\cite{CITEKEYS}.";
+
+        var thanks = "We thank the Computational Infrastructure for Geodynamics (geodynamics.org) which is funded by the National Science Foundation under award EAR-0949446 and EAR-1550901 for supporting the development of ASPECT.";
+    </script>
+
+    <p><b>Please select the following features you used:</b>
+<br/>
+To give credit to the authors of significant pieces contributed to ASPECT, please select the version and check the following checkboxes of features that you feel were important to get your computational results. This will add relevant citations to the list of papers below:
+</p>
+
+    <form action="" class="editbox">
+        <label>Version: <select id="ver" onchange="update()">
+	    <option value="dev">dev</option>
+	    <option value="2.0.1">2.0.1</option>
+	    <option value="2.0.0">2.0.0</option>
+	    <option value="1.5.0">1.5.0</option>
+	    <option value="1.4.0">1.4.0</option>
+	</select></label>
+	<br/><br/>
+        <label><input type="checkbox" id="fs" onchange="update()">Free Surface</label>
+        <label><input type="checkbox" id="melt" onchange="update()">Melt Transport</label>
+        <label><input type="checkbox" id="particles" onchange="update()">Particles</label>
+        <label><input type="checkbox" id="dg" onchange="update()">DG advection</label>
+	&nbsp;
+        <label><input type="checkbox" id="checkall" onchange="do_checkall()">(check all)</label>
+	<br/>
+	<!-- TODO: plasticity? Newton? vof? geoid? topography? -->
+    </form>
+
+    <p><b>Please use the following notation to cite ASPECT:</b></p>
+    <div id="intext" class="box"></div>
+
+    <p><b>Please add the following Acknowledgement:</b></p>
+    <div id="thanks" class="box"></div>
+
+    <p><b>Papers:</b>
+<br>
+The canonical reference for ASPECT is the first publication listed here, but please consider citing
+all the references below:
+</p>
+    <div id="paper" class="box"></div>
+
+    <p><b>Bibtex:</b></p>
+    <div id="bibtex" class="box"></div>
+
+
+    <script type="text/javascript">
+        function do_checkall() {
```

Comment:
there's not a line of comment in the following 80 lines of code :-( I don't think I really understand what's happening here.